container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -328,10 +329,28 @@ function run_git_tempest {
     tempest init openshift
     pushd $TEMPEST_DIR
 
-    discover_tempest_config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES} \
+    # We're running cleanup only under certain circumstances
+    if [[ ${TEMPEST_CLEANUP} == true ]]; then
+        # generate a simple tempest.conf so that we can run --init-saved-state
+        discover-tempest-config
+        # let's remove the images that discover-tempest-config creates by default
+        # so that the're not part of the saved_state.json and can be deleted
+        # by tempest cleanup later
+        openstack image list -c Name -f value | grep cirros | xargs -I {} openstack image delete {}
+        tempest cleanup --init-saved-state
+    fi
+
+    upload_extra_images
+
+    discover-tempest-config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES} \
```

Comment:
```suggestion
    discover_tempest_config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES} \
```

Reply:
thanks! i totally missed the - vs _ change